### PR TITLE
fix rollout and game struct err

### PR DESCRIPTION
**Pull request by ls-2018**
- Fixed struct field errors

### CODE REVIEW

File: versioned_docs/version-v1.5/user-manuals/uniteddeployment.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ spec:
 ...
 ```
 
-## Capacity Planning For Subsets (MaxReplicas)
+## Capacity Planning For Subsets (replicas)
 
 **FEATURE STATE:** Kruise v1.5.1
 
-UnitedDeployment offer the option to define the `MaxReplicas` for each subset, allowing you to effectively manage your resource allocation.
+UnitedDeployment offer the option to define the `replicas` for each subset, allowing you to effectively manage your resource allocation.
```
Comment (furykerry) [inline versioned_docs/version-v1.5/user-manuals/uniteddeployment.md:90]: MaxReplicas is a new field in Kruise v1.5.1, so it is not an error

**Review by chrisliu1995 — COMMENTED** [review]
/LGTM

**Pull request merged**